Assets/Scripts/FollowingCamera.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
```

Comment:
We do have a script already for a following camera. Not sure if having an externally controlled camera is good? We could just make appropriate camera render to different viewports if we want to have different following cameras for different players.